ui/src/app/pages/workspace/unavailable-tier-modal.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import {Button} from 'app/components/buttons';
+import {Modal, ModalBody, ModalFooter, ModalTitle} from 'app/components/modals';
+import {displayNameForTier} from 'app/utils/access-tiers';
+
+interface Props {
+  accessTierShortName: string;
+  onCancel: Function;
+}
+
+export const UnavailableTierModal = (props: Props) => {
+  const {accessTierShortName, onCancel} = props;
+  return <Modal onRequestClose={() => onCancel()}>
```

Comment:
Any reason why not:
```suggestion
  return <Modal onRequestClose={onCancel}>
```

Reply:
Because we don't have confidence yet that we aren't using `this` in hazardous ways within the codebase.